web/posts/2018-07-29-how-i-use-stuff-day-one.markdown

```diff
@@ -0,0 +1,184 @@
+previous year, and to see how they have changed. Also entertaining are the
+times when I think of an incredibly clever answer only to realize it's the
+_exact_ same thing I wrote in a previous year.
+
+### Productivity (154 Entries)
+There are two types of entries that go into this journal: Daily and Weekly
+productivity entries. These entries are similar to my Q&A entries, except that
+they ask the same questions each time:
+
+#### Daily:
+* What was something important that you accomplished today?
+* What is one thing you're thankful for today?
+
+#### Weekly:
+* What were your accomplishments, highlights, and wins from the week?
+* What were your failures from the past week, and how could you improve on them?
+* What were your loose ends for the week?
+* What did you learn or discover this week?
+* How are you coming along with your overarching big goals and projects?
+
+I'm prompted to write these as a part of my daily and weekly reviews. I have
+[Workflows] set up for guiding me through the prompts and adding them to the
+right journal with the right default tags:
+
+[Workflows]: https://www.workflow.is/
+
+ * [Daily Journal Entry workflow](https://workflow.is/workflows/0881f818e77048c4a310e8d853237505)
+ * [Weekly Journal Entry workflow](https://workflow.is/workflows/52e62dc773544e1fae78f24507a17b58)
+
+This is the newest addition to my practice, and so is the one I'm still
+refining. I don't remember what prompted me to start these, but I'm on my 4th
+month of writing them, and I've been enjoying it. Initially, these entries
+lived in my main journal, but once I decided that I wanted to make it a
+regular practice, I created a new journal to hold them.
+
+Writing these has been a nice addition to my practice. The idea is akin to
+gratitude journaling, but calling it "Productivity" journaling helps me keep
+from getting all fluffy with the entries. It's a nice way to regularly remind
+myself of the things I'm getting done and the things I'm enjoying.  The weekly
+entries give me a chance to reflect on my successes/failures from the week and
+helps me come up with a plan to continue/fix them.
+
+### Home Screens (78 Entries)
+Every month I have a recurring task to take a screenshot of my home screens
+for my iPad, iPhone, and Apple Watch and dump them in here (with an
+appropriate tag). I love the idea of keeping a monthly record of what apps
+I've allowed to live on my home screen. It’s also nice to relive these old
+setups on the first of each month, when “On This Day” surfaces them again.
+
+### Social (32 Entries)
+The entry count here is artificially low because I _still_ haven't gotten
+around to importing my backlog, but this is essentially a local copy of [my
+microblog](https://micro.gordonfontenot.com). It's currently being populated
+via [an RSS trigger through IFTTT][microblog-ifttt]. I wish that Day One had
+an API so that I could hit this directly, but until that exists, this will
+have to do.
```

Comment:
When you publish this, you should send it to Manton, maybe via the slack channel. He has posted links to these kinds of posts before.